### ISSUE TITLE
force_unwrapping not recognising force unwraps in return statements using subscript.

### ISSUE THREAD

**Issue by lennartstolz** [realm/SwiftLint#813]
Hey ho! 👻 

It looks like the `force_unwrapping` rule doesn't recognise force unwraps in return statements.

Have a look at the following piece of code:

````swift
let dict: [String: String] = [
    "Foo": "Bla",
    "Boooo": "👻"
]
// This force_unwrapping creates a warning.
let hasWarning: String = dict["Boooo"]!

func bla() -> String {
    // This force_unwrapping doesn't create a warning.
    return dict["Boooo"]!
}

print(bla())
````

The first unwrapping creates a warning the second one not.

The `force_unwrapping` rule is enabled.

````bash
+--------------------------------+--------+-------------+------------------------+--------------------------------------------------+
| identifier                     | opt-in | correctable | enabled in your config | configuration                                    |
+--------------------------------+--------+-------------+------------------------+--------------------------------------------------+
| force_unwrapping               | yes    | no          | yes                    | warning                                          |
+--------------------------------+--------+-------------+------------------------+--------------------------------------------------+
````

I'm running version *0.12.0*.
